docs/console.md

```
@@ -0,0 +1,129 @@
+## yii scaffold/status
+
+Reads `scaffold-lock.json` and compares the recorded hash of every tracked file to its current
+on-disk hash. Outputs a status table.
+
+```bash
+yii scaffold/status
+```
+
+Example output:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)